README.md

```diff
@@ -40,6 +40,22 @@
 
   The Language Server for this client can be [found here](https://github.com/angelozerr/lsp4xml).
 
+## Supported VS Code settings
+
+The following settings are supported:
+
+* `xml.trace.server` : Allow user to view RPC communication through Output tab.
+* `xml.catalogs` : Indicate XML catalog files
+* `xml.logs.client` : Enable/disable logging to client output.
+* `xml.fileAssociations` : Allow user to associate an XML file pattern with a XML Schema to use.
+* `xml.format.splitAttributes` : A tag's attributes get formatted onto multiple lines.
```

Comment:
Set to `true` to split node attributes onto multiple lines during formatting. Defaults to `false`.